akka-actor/src/main/scala/akka/actor/ActorSystem.scala

```diff
@@ -170,18 +170,6 @@ object ActorSystem {
 
   val Version: String = akka.Version.current // generated file
 
-  val EnvHome: Option[String] = System.getenv("AKKA_HOME") match {
-    case null | "" | "." => None
-    case value           => Some(value)
-  }
-
-  val SystemHome: Option[String] = System.getProperty("akka.home") match {
-    case null | "" => None
-    case value     => Some(value)
-  }
-
-  val GlobalHome: Option[String] = SystemHome.orElse(EnvHome)
-
```

Comment:
Oh, they actually was still in master, I missed that somehow and thought the forum question was about old sources.